example/etcd/etcd-kind.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
```

Comment:
is this etcd kind example sourced from somewhere? Could we link the source if so?

Reply:
It's our own thing. We have some static manifests for etcd in this directory, and I just took those and added a storageclass.